samples/sensor/lm73c/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(lm73c)
+
+FILE(GLOB app_sources src/*.c)
+target_sources(app PRIVATE ${app_sources})
```

Comment:
Please add the driver and application in two separate commits

Reply:
OK, will pull this one and do separate, I guess driver has to go first and be merged?

Reply:
Separate commits in the same PR. Please see the [Commit Gudelines](https://docs.zephyrproject.org/latest/contribute/index.html#commit-guidelines)

Reply:
OK